python/amici/petab_import.py

```diff
@@ -20,42 +21,51 @@
 logger = logging.getLogger(__name__)
 
 
-def get_fixed_parameters(condition_df: pd.DataFrame,
-                         sbml_model: 'libsbml.Model',
-                         const_species_to_parameters: bool = False):
+def get_fixed_parameters(
+        sbml_model: 'libsbml.Model',
+        condition_df: Optional[pd.DataFrame] = None,
+        const_species_to_parameters: bool = False) -> List[str]:
```

Comment:
why does the condition_df get an Optional and the const_species_to_parameters not?

Reply:
Because `Optional` does not mean, that there is a default value, but that `None` is a valid option.